includes/wp-admin/class-settings.php

```diff
@@ -389,60 +389,152 @@ public static function add_settings_help_tab() {
+					'</figure>' . "\n" .
+					'<h4>' . \esc_html__( 'Usage Tips', 'activitypub' ) . '</h4>' . "\n" .
+					'<p>' . \esc_html__( 'This block works well on community pages, about pages, or sidebars. The compact style is ideal for sidebars with limited space, while the Lines style provides clear visual separation between followers in wider layouts.', 'activitypub' ) . '</p>' . "\n" .
+					'<p>' . \esc_html__( 'The block includes pagination controls when you have more followers than the per-page setting, allowing visitors to browse through all your followers.', 'activitypub' ) . '</p>' . "\n" .
+
+					'<h2>' . \esc_html__( 'Fediverse Reactions', 'activitypub' ) . '</h2>' . "\n" .
+					'<p>' . \esc_html__( 'This block displays likes and reposts from the Fediverse for your content, showing engagement metrics from across federated networks.', 'activitypub' ) . '</p>' . "\n" .
+					'<figure class="activitypub-block-screenshot">' . "\n" .
+					'<img src="' . \esc_url( ACTIVITYPUB_PLUGIN_URL . 'assets/img/reactions.png' ) . '" alt="' . \esc_attr__( 'Fediverse Reactions block', 'activitypub' ) . '" width="600" height="auto">' . "\n" .
+					'<figcaption class="activitypub-screenshot-caption">' . \esc_html__( 'The Reactions block showing likes and reposts from the Fediverse.', 'activitypub' ) . '</figcaption>' . "\n" .
+					'</figure>' . "\n" .
+					'<h4>' . \esc_html__( 'How It Works', 'activitypub' ) . '</h4>' . "\n" .
+					'<p>' . \esc_html__( 'The Reactions block dynamically fetches and displays likes and reposts (boosts) that your content receives from across the Fediverse. It updates automatically as new reactions come in, providing real-time feedback on how your content is being received.', 'activitypub' ) . '</p>' . "\n" .
+					'<h4>' . \esc_html__( 'Usage Tips', 'activitypub' ) . '</h4>' . "\n" .
+					'<p>' . \esc_html__( 'This block provides social proof by showing how your content is being received across the Fediverse. It works best at the end of posts or pages where it can display engagement metrics without interrupting the content flow.', 'activitypub' ) . '</p>' . "\n" .
+
+					'<h2>' . \esc_html__( 'Federated Reply', 'activitypub' ) . '</h2>' . "\n" .
+					'<p>' . \esc_html__( 'This block allows you to respond to posts, notes, videos, and other content on the Fediverse directly within your WordPress posts.', 'activitypub' ) . '</p>' . "\n" .
+					'<figure class="activitypub-block-screenshot">' . "\n" .
+					'<img src="' . \esc_url( ACTIVITYPUB_PLUGIN_URL . 'assets/img/reply.png' ) . '" alt="' . \esc_attr__( 'Federated Reply block', 'activitypub' ) . '" width="600" height="auto">' . "\n" .
+					'<figcaption class="activitypub-screenshot-caption">' . \esc_html__( 'The Federated Reply block with embedded Fediverse content and reply interface.', 'activitypub' ) . '</figcaption>' . "\n" .
+					'</figure>' . "\n" .
```

Comment:
Let's mention the bookmarklet here and link to the tools site!

Reply:
Fixed in 445704a2515d27a6cda6f0a76aa326a88f673336